### DISCUSSION TITLE
get error in slim theme

### DISCUSSION THREAD

**Discussion by aerosadegh** [JanDeDobbeleer/oh-my-posh#3027] [Troubleshoot] [answered]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I do update to `12.9.1` and get error in slim theme:

![image](https://user-images.githubusercontent.com/5039032/198814638-31d62ca6-a62f-48d4-b9f4-45849a804fb4.png)

### Theme

slim.omp.json

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
!!text property not defined!!
```

**Comment by JanDeDobbeleer** [comment, marked as answer]
@aerosadegh this probably implies that the theme got updated, but oh-my-posh is still an older version. Can you validate that your oh-my-posh executable is also `12.9.1`? That error message is no longer (for a long time now) in the codebase.

**Reply by aerosadegh** [reply]
Yes, I check multiple times. I do install manually.
One thing is seems wrong; The themes on git repo was difference with that installed!

for example in slim:
Git:
   
![image](https://user-images.githubusercontent.com/5039032/198862376-308f1b62-f635-4021-8e84-7b9bc3d9ce9b.png)

Installed:

![image](https://user-images.githubusercontent.com/5039032/198862409-7cccb25a-e53a-4993-95c3-6f4f86c6009a.png)


Where is the `template` key?


I do set a command in profile1.ps1 to set theme from my own directory. that raise this error. but after your answer test bellow command and worked properly!
![image](https://user-images.githubusercontent.com/5039032/198862528-78d97d10-f2f1-4a27-8a9a-92551b9a95be.png)

**Reply by JanDeDobbeleer** [reply]
@aerosadegh the installed location you show isn't the install location the installer creates. That's still a very old PowerShell module version.

**Reply by aerosadegh** [reply]
sorry but i get new issue for `slim` and `slimfat` theme

![image](https://user-images.githubusercontent.com/5039032/198867133-92908cec-e7b8-4e69-999a-a33deca838df.png)

Is that correct when I execute a command previous line is deleted? This is happened in  `slim` and `slimfat` theme.

**Reply by JanDeDobbeleer** [reply]
@aerosadegh yes, that's the [transient](https://ohmyposh.dev/docs/configuration/transient) prompt.

**Reply by aerosadegh** [reply]
Very thank you for this cool tool! ✌️

**Comment by TKIPisalegacycipher** [comment]
In my case the issue was using Set-Theme, which looks like it's been deprecated. The solution was to update my $PROFILE to use the init line in https://ohmyposh.dev/docs/installation/customize
